src/main/java/io/redlink/more/data/transformers/garmin/SleepSummaryTransformer.java

```diff
@@ -0,0 +1,50 @@
+package io.redlink.more.data.transformers.garmin;
+
+import io.redlink.more.data.custom.model.GarminDataPoint;
+import io.redlink.more.data.model.DataPoint;
+import io.redlink.more.data.model.DataType;
+import io.redlink.more.data.model.Observation;
+import io.redlink.more.data.model.garmin.GarminSummaryType;
+import io.redlink.more.data.model.garmin.transformation.GarminSleepData;
+import io.redlink.more.data.model.garmin.transformation.GarminTimeData;
+import io.redlink.more.data.util.MapperUtils;
+import org.apache.commons.lang3.Range;
+import org.springframework.stereotype.Component;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class SleepSummaryTransformer extends AbstractGarminTransformer {
+    @Override
+    public GarminSummaryType getSupportedType() {
+        return GarminSummaryType.SLEEPS;
+    }
+
+    @Override
+    protected List<DataPoint> transformToDataPoint(List<Observation> observations, GarminDataPoint garminDataPoint) {
+        var sleepData = dataToSleep(garminDataPoint);
+        return sleepData.entrySet().stream().flatMap(entry ->
+                        transformGarminTimeDataToDataPoint(
+                                observations,
+                                garminDataPoint.getSummaryId(),
+                                entry.getKey(),
+                                entry.getValue()
+                        ).stream())
+                .toList();
+    }
+
+    @Override
+    protected List<DataPoint> filterDataPointByTimeRange(List<Range<Instant>> validTimeRanges, List<DataPoint> dataBulk) {
+        return dataBulk;
```

Comment:
Ich gehe mal davon aus, dass das noch nicht implementiert ist, oder?

```suggestion
        return dataBulk; //FIXME: implement
```
Ansonsten würde ich einen Kommentar anfügen, dass das filtern unnötig ist, da die Filterung auf GarminDataPoint ebene ausreicht

```suggestion
        return dataBulk; //NOTE: filtering on GraminDataPoint level is sufficient
```